File: leetCode/2023/June/count-negative-numbers-in-a-sorted-matrix.js
/**
 * @param {number[][]} grid
 * @return {number}
 */
//https://leetcode.com/problems/count-negative-numbers-in-a-sorted-matrix/description/
var countNegatives = function(grid) {
    let count  = 0;

    grid.forEach(g => {
        g.forEach(i => {
            if(i < 0) {
                count++;
            }
        })
    })

    return count;
};

File: leetCode/2023/June/count-negative-numbers-in-a-sorted-matrix.rs
impl Solution {
    pub fn count_negatives(grid: Vec<Vec<i32>>) -> i32 {
        let mut count = 0;
        for row in grid{
            for col in row {
                if col < 0 {
                    count += 1;
                }
            }
        }

        return count;
    }
}